.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+.github/CODEOWNERS @sonarsource/sonarqube-team
```

Comment:
.github/CODEOWNERS @SonarSource/languages-team-dotnet @duncanp-sonar

Reply:
I already created a PR to  set the codeowners. You can see it here: https://github.com/SonarSource/sonar-scanner-msbuild/pull/1205
As soon as the PR gets green, I will merge it.